examples/first-frame/CMakeLists.txt

```diff
@@ -28,3 +28,5 @@ if( WIN32 )
         )
     endif()
 endif()
+
+target_link_libraries(${PROJECT_NAME} PRIVATE aditof command_parser)
```

Comment:
first-frame already links to "aditof" (see line 8). You should add command_parser there as well.

Reply:
Solved